SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java

```diff
@@ -74,41 +74,52 @@ public void setListener(SEPADirectDebitListener listener) {
      * @param sepaDirectDebitRequest the {@link SEPADirectDebitRequest}.
      */
     public void tokenize(final FragmentActivity activity, final SEPADirectDebitRequest sepaDirectDebitRequest) {
+        braintreeClient.sendAnalyticsEvent("sepa-direct-debit.selected.started");
```

Comment:
Open to suggestions on a better analytics event for the start of the entire flow (we didn't want to use tokenize since that is the name of a specific API call during the flow as well).